src/modules/user-list.tsx

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from '@apollo/client';
+import React from 'react';
+import { Cell } from '../components/cell';
+import { UsersListData, USERS_QUERY } from '../data/users-query';
+
+export const UserList: React.FC = () => {
+  const token = localStorage.getItem('token');
+  const { data, error } = useQuery<UsersListData>(USERS_QUERY, {
+    context: {
+      headers: {
+        Authorization: token,
+      },
+    },
+    variables: {
+      offset: 0,
+      limit: 10,
+    },
+  });
+
+  const list = data?.users.nodes.map((user) => (
+    <Cell key={user.id}>
+      <p>
+        {user.name} : {user.email}
```

Comment:
Em casos assim, é preferível deixar isso dentro de `, porque se você precisar colocar um espaço ou algo nesse sentido, o react corta.
```suggestion
        {`${user.name} : ${user.email}`}
```